pkg/dataplane/http/context.go

```diff
@@ -434,26 +439,35 @@ func (c *context) UpdateItemSync(updateItemInput *v3io.UpdateItemInput) error {
 			body["UpdateMode"] = updateItemInput.UpdateMode
 		}
 
-		_, err = c.putItem(&updateItemInput.DataPlaneInput,
+		response, err = c.putItem(&updateItemInput.DataPlaneInput,
 			updateItemInput.Path,
 			putItemFunctionName,
 			updateItemInput.Attributes,
 			updateItemInput.Condition,
 			putItemHeaders,
 			body)
 
+		mtimeSecs, mtimeNSecs := parseMtimeHeader(response)
+		output := &v3io.UpdateItemOutput{MtimeSecs: mtimeSecs, MtimeNSecs: mtimeNSecs}
```

Comment:
Why do you need this "output" variable? Just set .Output to `&v3io.UpdateItemOutput{...`
Same as in the other place(s)